.environment/eslint/run-eslintCheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+function note() {
+    echo "eslint-check> info: ${*}"
+}
+
+function js_fmt_check() {
+    note "Checking JavaScript formatting."
+    MODIFIED_TSX_FILES_COUNT=$(git status --porcelain | grep "\.tsx$" | wc -l)
+    MODIFIED_JS_FILES_COUNT=$(git status --porcelain | grep "\.js$" | wc -l)
```

Comment:
Do we want to limit to the `/src` directory explicitly for these greps? 

I'm wondering if we might hit future issues with some files in `/cypress` or `/public`? Right now it's fine, just trying to think to potential future gotchas. 

It's fine to not change this, just thought I'd mention it as a possibility.